Dockerfile

```diff
@@ -7,11 +7,6 @@ RUN apt-get update && \
 # Create appuser
 RUN useradd -ms /bin/bash appuser
 
-# Get exercism's tooling_webserver
-RUN curl -L -o /usr/local/bin/tooling_webserver \
-  https://github.com/exercism/tooling-webserver/releases/download/0.10.0/tooling_webserver && \
-  chmod +x /usr/local/bin/tooling_webserver
```

Comment:
@iHiD Is it safe to remove this from the test runner?

Reply:
Yes